src/OrchardCore/OrchardCore/Localization/ICalendarManager.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NodaTime;
+
+namespace OrchardCore.Localization
+{
+    public interface ICalendarManager
+    {
+        IEnumerable<string> GetCalendars();
+        Task<CalendarSystem> GetCurrentCalendar();
+        CalendarSystem GetCalendarByName(string calendarName);
```

Comment:
I think you can remove `GetCalendarByName` and keep it private in the implementation.